app/views/declaration/summary/item_section.scala.html

```diff
@@ -28,10 +28,10 @@
         supporting_documents: supporting_documents
 )
 
-@(mode: Mode, item: ExportItem)(implicit messages: Messages, journeyRequest: JourneyRequest[_])
+@(mode: Mode, item: ExportItem, actionsEnabled: Boolean = true)(implicit messages: Messages, journeyRequest: JourneyRequest[_])
```

Comment:
What's the new 'actionsEnabled' param for?

Reply:
So this is doing the same thing as passing `Mode.Print`, but I did not want to use the mode for the purpose of remove page.

Reply:
I see.  Didn't notice that you were re-using the item_summary for your new remove_item page.

Might be useful when we re-use these summary components to "preview" a dec we are about to clone.

Reply:
Is it worth adding new `Mode.Preview` ?

Reply:
Could maybe rename Mode.Print -> Mode.Preview.  Then you could use it for item_remove summary?  (And get rid of your new actionsEnabled parameter?

Reply:
Sounds good! I'd really prefer doing it this way. Is "Preview" a good word for all use cases?

Reply:
On second thoughts, is this just an abuse of the "Mode".    Maybe it would be better to get rid of Mode.Print and implement your `actionsEnabled` flag everywhere.

Otherwise on your item_remove page you are going to be ignoring the Mode passed into the page and pass Mode.Preview (or whatever) into the item_section.

My fault really.  I should have done it properly when I did the declaration print view.